sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOITHelper.java

```diff
@@ -36,4 +42,55 @@ public static String getHashForRecordCount(int recordCount, Map<Integer, String>
     }
     return hash;
   }
+
+  public static void retry(RetryFunction function, int attempts, long delay)
```

Comment:
Can we pass a lambda instead of introducing a new interface RetryFunction ?

Reply:
Well, I know this looks a little bit messy but I find it most `clear` way to do this.
There are two other options:
* Callable - it requires to have result in return. That way for example in JdbcIOIT, the method `createTable` has to return Void. In my opinion this looks worse:
```
Void createTable() throws SQLException {
    DatabaseTestHelper.createTable(...);
    return null;
  }
```
* Runnable - this is from old Java and it doesn't throws Exceptions. In that case if any developer who would like use our `retry()` will have to surround body of lambda with try catch.

In current state, the `RetryFunction` here is our custom `Runnable` which throws Exception and doesn't require result message.

For now I can't find any better solution. 
Maybe cleaner will be moving interface to `IOITHelper`? What do you think?